guides/common/modules/con_provisioning-methods-in-projectname.adoc

```diff
@@ -0,0 +1,34 @@
+[id="provisioning-methods-in-{ProjectNameID}_{context}"]
+= Provisioning Methods in {ProjectName}
+
+Provisioning methods are part of the {ProjectName} application life cycle to create, manage, and update hosts.
```

Comment:
```suggestion
Provisioning methods are part of the {ProjectName} application lifecycle to create, manage, and update hosts.
```

This does not make any sense to me. Please reword and split "Project lifecycle" and "lifecycle of managed hosts" more clearly.

Reply:
I think I found a way to introduce the module without using this weird sentence. So, addressed.